bentoml/_internal/frameworks/tensorflow_v2.py

```diff
@@ -250,7 +247,9 @@ def _run_method(
             with tf.device(runnable_self.device_name):
 
                 def _mapping(item: "TFArgType") -> "tf_ext.TensorLike":
```

Comment:
shall we move this function definition out of the inference path? Or it will create the function for each request

Reply:
@larme could you do that in a follow-up PR?